docs/developer/protocol/deployments/index.md

```diff
@@ -0,0 +1,35 @@
+---
+id: deployments
+title: Deployments
+category: subpage
+contributors: <Jeroen:jeroen@k-f.co>
+---
+
+# Deployments
+
+## Smart contracts
+
+Centrifuge V3 is not yet deployed on mainnet.
+
+Testnet deployments can be found here: https://github.com/centrifuge/protocol-v3/tree/post-electisec/env
+
+## Centrifuge IDs
+
+The protocol uses `centrifugeId` (`uint16`) as an identifier of the network. The following IDs are used:
+
+
+### Mainnet
+
+| Network          | `centrifugeId` |
+|------------------|----|
+| Ethereum Mainnet | 1  |
+| Base             | 2  |
+| Arbitrum         | 3  |
+
+### Testnet
+
+| Network          | `centrifugeId` |
+|------------------|----|
+| Ethereum Sepolia | 1  |
+| Base Sepolia     | 2  |
+| Arbitrum Sepolia | 3  |
```

Comment:
CR

I think we should use different ids for testnets. That is the case for most bridges too and makes misconfiguration much harder. Wdyt?